redeem/gcodes/M2x.py

```diff
@@ -399,19 +369,16 @@ def get_formatted_description(self):
 
 class M27(M2X):
   def execute(self, g):
-
     line_position, byte_position = self.printer.sd_card_manager.get_position()
     size_lines, size_bytes = self.printer.sd_card_manager.get_file_size()
     file_name = self.printer.sd_card_manager.get_file_name()
-
     # message to inform that we have completed the print
-    active = self.printer.sd_card_manager.get_status()
-    if active:
-      message = "SD printing byte {}/{}".format(byte_position, size_bytes)
-    else:
+    if file_name is None:
```

Comment:
Isn't this set of changes already present in #145 ? Why are we not merging that, then you can build your changes on top of that?

Makes no sense that you're forcefully deprecating my PR while doing this. It's not the way I was instructed to use Git.

Reply:
Your PR is not mergeable. And, I have also corrected the code.

Reply:
If you wish to adopt the revised code and present it as a fast-forward merge to the present head, then go ahead and do so. As I told @ThatWileyGuy, I don't care about PRs. To me they just get in the way. It is the individual commits that matter. And this one is still attributed to you.

Reply:
And, BTW, "building on" to correct something is not a good practice. If it ain't right, it's not ready to be merged.